onto_spread_ed/templates/base.html

```diff
@@ -55,7 +55,7 @@
         <ul class="navbar-nav me-auto mb-2 mb-lg-0">
         <li class="nav-item"><a class="nav-link active" aria-current="page" href="/">Home</a></li>
                     {% if login is defined and config["USERS_METADATA"][login]["admin"] %}
-                    	<li class="nav-item"><a class="nav-link active" aria-current="page" href="/admin">Admin</a></li>
+                        <li class="nav-item"><a class="nav-link active" aria-current="page" href="/admin">Admin</a></li>
```

Comment:
This should use the prefix as well.

Reply:
for this would it be: href="{{url_for('admin.dashboard')}}" ? 
I have this confusion because in the admin.py, there is already a url_prefix defined in the blueprint .
@b-gehrke Please let me know if href="{{url_for('admin.dashboard')}}"  is the right way to do it for this one.

This is a part of the code of admin.py:

bp = Blueprint("admin", __name__, url_prefix="/admin", template_folder="../templates/admin")

@bp.route("/")
@bp.route("/dashboard")
@verify_admin
def dashboard():
    return render_template("dashboard.html",

Reply:
Yes that is correct. The key is constructed using the first parameter of `Blueprint("admin", ...)` (here `"admin"` and the name of the function (here `"dashboard"`)